.github/workflows/bandit.yml

```diff
@@ -0,0 +1,51 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# Bandit is a security linter designed to find common security issues in Python code.
+# This action will run Bandit on your codebase.
+# The results of the scan will be found under the Security tab of your repository.
+
+# https://github.com/marketplace/actions/bandit-scan is ISC licensed, by abirismyname
+# https://pypi.org/project/bandit/ is Apache v2.0 licensed, by PyCQA
+
+name: Bandit
+on:
+  push:
+    branches: [ "master", "stable", feature-*, HOTFIX-* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+
+permissions: {}
+
+jobs:
+  bandit:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+    
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Bandit Scan
+        uses: shundor/python-bandit-scan@v1.0
```

Comment:
Works but the upstream repo uses codeql@v2 and needs to be updated if possible.

🛑 Please wait for PR to be pushed up-stream to fix this.

Reply:
```suggestion
        uses: reactive-firewall/python-bandit-scan@v2
```

Reply:
:+1: Ok same results from my v2 and previous upstream's v1.0 (as they should be if I did things right)

Now let's transition to v2.1 (mine after a dependabot auto-update) and see if the warnings are fixed.